src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubAuthenticationTests.kt

```diff
@@ -1,16 +1,23 @@
-package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.api.auth
+package org.vaccineimpact.orderlyweb.customConfigTests
```

Comment:
Since the default auth provider is Montagu, these tests of GitHub auth had to be moved to `customconfigtests`